app/src/main/java/com/matanh/transfer/server/KtorServer.kt

```diff
@@ -528,34 +562,50 @@ fun Application.ktorServer(
                 }
             }
 
-            // HTTP Interface
+            // HTTP Interface - optimized for large file uploads
```

Comment:
`optimized for large file uploads` - the changes are generic try/catch, and replace the string `file` to `Large file`.